packages/forms-web-app/src/controllers/projects/documents/controller.js

```diff
@@ -21,13 +21,15 @@ const getApplicationDocuments = async (req, res) => {
 		const pageFeatureToggles = featureToggles();
 		const pageDataObj = await pageData(case_ref);
 
-		const { documents, filters, pagination } = await searchDocuments(case_ref, query);
+		const { documents, examinationLibraryDocumentHtml, filters, pagination } =
```

Comment:
This doesn't look right. 

This is returning data in raw form not transforming anything. 

By putting the html conversion in here it's now doing more than one thing. 

Plus you have added conditional logic into something that should be a simple call to get the data. 

get the examination response from here. 

And in get documents do the html conversions. 

This separation is important for extendability. We want one place to get the response (responses) and the subsequent functions to manipulate the shape of the data.

Reply:
with the documents get you would add this extra documents to the document object for the view. 

`documents: { documentsView, examinationLibraryDocumentHtml}`'

this way anything to do with documents in the view is scoped to that key.